tensorflow/core/BUILD

```diff
@@ -1762,6 +1762,9 @@ tf_cc_tests(
         "graph/graph_test.cc",
         "graph/node_builder_test.cc",
         "graph/optimizer_cse_test.cc",
+# INTEL_MKL start
```

Comment:
Can you change this to conditionally include this file, only if MKL is enabled? Example:

https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/BUILD#L796